compiler-rt/cmake/Modules/AllSupportedArchDefs.cmake

```diff
@@ -59,20 +61,33 @@ else()
   set(ALL_LSAN_SUPPORTED_ARCH ${X86} ${X86_64} ${MIPS64} ${ARM64} ${ARM32}
       ${PPC64} ${S390X} ${RISCV64} ${HEXAGON} ${LOONGARCH64})
 endif()
-set(ALL_MSAN_SUPPORTED_ARCH ${X86_64} ${MIPS64} ${ARM64} ${PPC64} ${S390X}
-    ${LOONGARCH64})
+if (OS_NAME MATCHES "FreeBSD")
+  set(ALL_MSAN_SUPPORTED_ARCH ${X86_64} ${ARM64})
+else()
+  set(ALL_MSAN_SUPPORTED_ARCH ${X86_64} ${MIPS64} ${ARM64} ${PPC64} ${S390X}
+      ${LOONGARCH64})
+endif()
+
 set(ALL_HWASAN_SUPPORTED_ARCH ${X86_64} ${ARM64} ${RISCV64})
 set(ALL_MEMPROF_SUPPORTED_ARCH ${X86_64})
 set(ALL_PROFILE_SUPPORTED_ARCH ${X86} ${X86_64} ${ARM32} ${ARM64} ${PPC32} ${PPC64}
     ${MIPS32} ${MIPS64} ${S390X} ${SPARC} ${SPARCV9} ${HEXAGON}
     ${RISCV32} ${RISCV64} ${LOONGARCH64})
-set(ALL_TSAN_SUPPORTED_ARCH ${X86_64} ${MIPS64} ${ARM64} ${PPC64} ${S390X}
-    ${LOONGARCH64} ${RISCV64})
+if (OS_NAME MATCHES "FreeBSD")
+  set(ALL_TSAN_SUPPORTED_ARCH ${X86_64} ${MIPS64} ${ARM64})
```

Comment:
Is the tsan case meant to retain MIPS64?

Reply:
That's a good point, I just copied the values from FreeBSD::getSupportedSanitizers(), but MIPS64 is gone in recent FreeBSD, so might as well drop it here completely?

Reply:
I think so. mips64 is still supported in FreeBSD 13 which has about 2 more years before EOL, but I think it's quite likely that sanitizers are not being used/tested at all on FreeBSD/mips64.